### ISSUE TITLE
'TypeError: Cannot read property 'set' of undefined' when assigning a button value outside 1 thru 8

### ISSUE THREAD

**Issue by TheNathannator** [ejj28/mididrumhero#5]
```
Uncaught Exception:
TypeError: Cannot read property 'set' of undefined
   at vJoySetButton
(C:\Users\[user]\AppData\Local\Programs\mididrumhero\resources\app.asar\index.js:127:26)
   at NodeMidiInput.<anonymous>
(C:\Users\[user]\AppData\Local\Programs\mididrumhero\resources\app.asar\index.js:162:9)
   at NodeMidiInput.emit (events.js:210:5)
```
Assigning a 9th button causes the above exception every time it is pressed. The same happens for a 0th or -1st button.

![Error message](https://user-images.githubusercontent.com/29052821/78694752-5b588700-78ba-11ea-98fc-71053475f86b.png)

![Button list](https://user-images.githubusercontent.com/29052821/78696206-5e547700-78bc-11ea-81ec-ce2306617dbb.png)

**Comment by towner-10** [comment]
For now this setting must be changed in your vJoy settings, values must be between 1-128 and the default number of buttons is 8. I am adding a feature to alert users of invalid inputs.

**Comment by towner-10** [comment]
Fixed in 3d635ea6b7b5022adf05a14f0a1642c149646db6.
